testdata/config.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Can you embed this (using `//go:embed`) into the test file?

Reply:
The tests use this via the use of `DOCKER_CONFIG` environment variable, so using embed wouldn't work.